dagshub/data_engine/model/datasource.py

```diff
@@ -304,26 +305,44 @@ def visualize(self, **kwargs) -> "fo.Session":
     def fields(self) -> List[MetadataFieldSchema]:
         return self.source.metadata_fields
 
+    def annotate(self) -> Optional[str]:
+        """
+        Sends all datapoints in the datasource for annotation in Label Studio.
+        It's recommended to not send a huge amount of datapoints to be annotated at once, to avoid overloading
+        The Label Studio workspace.
+
+        :return: Link to open Label Studio in the browser
+        """
+        return self.all().annotate()
+
     def send_to_annotation(self):
         """
-        Sends all datapoints in the datasource for annotation to labelstudio
+        deprecated, see annotate()
         """
-        self.send_datapoints_to_annotation(self.all().entries)
+        return self.annotate()
 
-    def send_datapoints_to_annotation(self, datapoints: Union[List[Datapoint], List[Dict]],
-                                      open_project=True) -> Optional[str]:
+    def send_datapoints_to_annotation(self, datapoints: Union[List[Datapoint], QueryResult, List[Dict]],
+                                      open_project=True, ignore_warning=True) -> Optional[str]:
```

Comment:
I used `force=False` in args everywhere, so maybe for convention keep it as force here too.
Although I like ignore_warning more

Reply:
Oops this was a mistake.
I prefer `ignore_warning` since we might want to reserve `force` for something more fundamental
```suggestion
                                      open_project=True, ignore_warning=False) -> Optional[str]:
```

Reply:
Valid 💯